jme3-core/src/main/java/com/jme3/system/AppSettings.java

```diff
@@ -289,6 +289,7 @@
         defaults.put("SwapBuffers", true);
         defaults.put("OpenCL", false);
         defaults.put("OpenCLPlatformChooser", DefaultPlatformChooser.class.getName());
+        defaults.put("UseRetinaFrameBuffer", false);// MacOS spec
```

Comment:
This is just a opinion but I think the default should be true? I would imagine someone would like to use all this native behaviour by default (given it also works perfectly in jME by default). In GLFW this is by default true also.